external/googleapis/protolists/discoveryengine.list

```diff
@@ -11,6 +11,7 @@
 @com_google_googleapis//google/cloud/discoveryengine/v1:engine_service.proto
 @com_google_googleapis//google/cloud/discoveryengine/v1:import_config.proto
 @com_google_googleapis//google/cloud/discoveryengine/v1:purge_config.proto
+@com_google_googleapis//google/cloud/discoveryengine/v1:recommendation_service.proto
```

Comment:
That looks like a new service. Do you mind adding a tracking bug if that is indeed the case?

Reply:
That new service was the motivation for this PR. I will generate it.